### DISCUSSION TITLE
Get 系以外のUseCase について

### DISCUSSION THREAD

**Discussion by chippy-ao** [tatsutakein-jp/asis-app#354] [Ideas]
現状だと使用する側が、`AsyncValue<void>` になり、`await` などが使用できないと思います。
`Raw<Future<void>>` で定義することで、単純に `Future` として扱うことができますが、いかがでしょう！

https://github.com/tatsutakein-jp/asis-app/blob/230d32acb49de8932bee0045d3399b142055833c/core/domain/lib/src/use_case/quest/add_quest_use_case.dart#L7-L13

**Comment by tatsutakein** [comment]
LGTM です！
特に不都合ないと思うので変えちゃいましょう〜！

**Reply by chippy-ao** [reply]
#355
Issue 立てをしましたので、対応しておきます。
